specs/_common/team_spec.py

```diff
@@ -0,0 +1,48 @@
+import os
+import uuid
+from expects import expect, equal
+from mamba import before, description, it
+from sdcclient import SdSecureClient, SdMonitorClient
+from specs import be_successful_api_call
+
+TEAM_PREFIX_NAME = 'sysdig-sdk - '
+
+with description("Teams", "integration", "teams") as self:
+    with before.all:
+        self.secure_client = SdSecureClient(
+                sdc_url=os.getenv("SDC_SECURE_URL", "https://secure.sysdig.com"),
```

Comment:
Should we default to a fake url instead of production for the tests?

Reply:
I followed the convention. All tests have production urls.